torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1286,6 +1314,15 @@ RegisterOperators reg2({
           "aten::append( " decl_type "[](a!) self, " decl_type              \
           "(c) el) -> " decl_type "[](a!)",                                 \
           listAppend<Shared<c_type>, c_type::ElemType>),                    \
+      Operator(                                                             \
+          "aten::extend(" decl_type "[](a!) self, " decl_type               \
+          " [](b) other) -> ()",                                            \
```

Comment:
The `b` annotation here is unnecessary, since you're not doing a write or creating an alias. So it's fine to leave it blank (which is sugar for "this is a read-only, unique value")

Reply:
Will fix

Reply:
Addresssed